src/components/Navbar.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+import { Link } from "react-router-dom";
+import { getReq } from "../utils/api";
+
+const Navbar = () => {
+  const [topics, setTopics] = useState([
```

Comment:
I'm thinking that you'll do this on the next ticket but we should be grabbing this from the api. I would have the intial state as an empty array rather than a pre-set list of topics and then map through those to create the links.